src/components/header/DiscoverHeaderButton.js

```diff
@@ -50,9 +50,10 @@ export default function DiscoverHeaderButton() {
     navigate,
   ]);
 
-  const onLongPress = useCallback(() => navigate(Routes.CONNECTED_DAPPS), [
-    navigate,
-  ]);
+  const onLongPress = useCallback(
+    () => navigate(Routes.PROFILE_SHEET, { address: 'mikedemarais.eth' }),
```

Comment:
was this just meant for ease of testing?

Reply:
Yep! I might revert this change.